app/Http/Controllers/ImagesController.php

```diff
@@ -102,6 +104,17 @@ public function update($id, Request $request)
         $originalImage = $this->aws->storeImageData($originalImage->__toString(), $originalFilename);
         $editedImage = $this->aws->storeImageData((string) $editedImage, 'edited_' . $post->id);
 
+        if (config('features.glide')) {
```

Comment:
Should we have a separate feature flag here, so we can disable this on local environments?

Reply:
Or perhaps just check against `APP_ENV`, or whether or not Fastly config is empty?

Reply:
yea I can add that!